docs/index.html

```diff
@@ -288,20 +288,54 @@ <h2>#TwitterThrowback</h2>
     var submitButton = document.getElementById('submit-button');
     var loadingIndicator = document.getElementById('loading-indicator');
 </script>
+<script>
+    /**
+     * @function getMonthAndDay
+     * @description returns the Date API Friendly month and day 
+     * @param monthAndDayString a string containing the month and day separated by a space
+    */
+     function getMonthAndDay(monthAndDayString) {
+        const todayAsArray = [new Date().getMonth() + 1, new Date().getDate()];
+        if (monthAndDayString.split(' ').length < 2) {
+            monthAndDayString = monthAndDayString.concat(` ${todayAsArray[1]}`);
+        }
+        const dummyDate = new Date(monthAndDayString);
+        return [ dummyDate.getMonth() + 1, dummyDate.getDate()];
+    }
+</script>
 <script>
     // Prefill the value if the user is coming from a direct link (oldtweets.today/#username)
     // Normally, I'd want to do this using a path, but GitHub Pages doesn't support routing via paths
+    
+    
     let hash = window.location.hash.split('#')[1];
     if (hash != '' && hash != undefined) {
-        usernameInput.value = hash;
+        const startingPointOfDate = hash.indexOf('?');
```

Comment:
Can't we just check for query string?

If this hash is making things messy, we might get rid of it and just stick to `?username=`. It still irks me a bit that we have to use the hash fragment wrongly.

Reply:
Well the app supports both at the moment. I think getting rid of query strings may be a bit tricky. I think the hash is there because of push state.